paperqa/utils.py

```diff
@@ -345,6 +346,33 @@ def remove_substrings(target: str, substr_removal_list: Collection[str]) -> str:
     return target
 
 
+def mutate_acute_accents(text: str, replace: bool = False) -> str:
+    """
+    Replaces or removes acute accents in a string based on the boolean flag.
+
+    If 'replace' is True, acute accents on vowels are replaced with an apostrophe (e.g., "á" becomes "'a").
+    If 'replace' is False, all acute accents are removed from the string.
+
+    Args:
+        text: The input string.
+        replace: A flag to determine whether to replace (True) or remove (False) acute accents.
```

Comment:
Can you combine the `replace` instructions into the `Args`? Unless I am missing something, it seems good to consolidate